lib/pbench/agent/toolmetadata.py

```diff
@@ -69,7 +69,7 @@ def __dataCheck(self):
         if not self.data:
             self.data == self.__getInitialData()
             if not self.data:
-                self.logger.error(f"Unable to access data through {self.mode}")
+                self.logger.error("Unable to access data through %s", self.mode)
```

Comment:
With multiple pre-formatted `info` and `warning` log messages above, for an `error` you change to post-formatted? Interesting choice! (No, I get that you're using the previous strings twice and there's no need for this to be pre-formatted; there's also no particular value in post-formatting since we're never going to disable `error` logs.)

I guess I'm just saying that I notice the break in pattern here. ;-)